### PR TITLE
Update CoreDNS image to in release 1.9 to gcr.io

### PR DESCRIPTION
**Pull request by rajansandeep**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Update the CoreDNS image to gcr.io in release 1.9

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Partially fixes https://github.com/kubernetes/kubernetes/issues/66460 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/addons/dns/coredns.yaml.sed [hunk 1]
```diff
@@ -95,7 +95,7 @@ spec:
           operator: "Exists"
       containers:
       - name: coredns
-        image: coredns/coredns:1.0.1
+        image: k8s.gcr.io/coredns:1.0.1
```
Comment (timothysc) [inline cluster/addons/dns/coredns.yaml.sed]: This seems like an older coreDNS version.

**Review by timothysc — APPROVED** [review]
/approve 
/lgtm

**Review by MrHohn — APPROVED** [review]
/approve

**Comment by MrHohn** [comment]
> Partially fixes #66460

@rajansandeep  What is the remaining steps to fix that?

**Comment by rajansandeep** [comment]
> What is the remaining steps to fix that?

@MrHohn This PR and https://github.com/kubernetes/kubernetes/pull/70374 will complete the fix.

File: cluster/addons/dns/coredns.yaml.sed (hunk 1 above)
Comment (rajansandeep) [reply cluster/addons/dns/coredns.yaml.sed]: Yes, this is the version for k8s release 1.9

**Comment by timothysc** [comment]
@kubernetes/sig-release-members  
cherry pick for 1.9

**Comment by liggitt** [comment]
no additional 1.9 releases are planned. can this be closed?

**Comment by rajansandeep** [comment]
@liggitt closing this.
/close

**Pull request closed without merging**